### ISSUE TITLE
Mocha isn't as dark as the preview

### ISSUE THREAD

**Issue by FrostedVolcano** [catppuccin/qbittorrent#26]
This is how the preview looks
![image](https://github.com/user-attachments/assets/ff444fa2-3d32-41be-9559-8a7bc75d2374)

And this is how it looks in my case
![image](https://github.com/user-attachments/assets/41f4e7b9-62c1-4a89-92c6-fe36183be7a7)

The preview seems to be darker.

**Comment by sgoudham** [comment]
Hey :wave:,

Are you using the files from the [latest release (v2.0.1)](https://github.com/catppuccin/qbittorrent/releases/tag/v2.0.1) of the theme? It's worth noting that I'd consider your screenshots to actually align more with Catppuccin with `base` taking up more of the background and `mantle` taking up just the search box.

I made the the decision to slightly go against our style guide for consistency reasons in the underlying framework. I wonder if these differences are just from the fact that my screenshots were taken on Linux and you seem to be running Windows? 

I don't have a Windows machine at the moment ready to test this unfortunately. @arminius-smh do you happen run Windows to test this out?

**Comment by arminius-smh** [comment]
![qbit_win_mocha](https://github.com/user-attachments/assets/e336ed61-0c2f-45f8-a393-7a718d16dcea)
 
> do you happen run Windows to test this out?

Suure, but unfortunately I can't reproduce it, in the image I am using the mocha theme from the latest release: https://github.com/catppuccin/qbittorrent/releases/download/v2.0.1/catppuccin-mocha.qbtheme, which is consistent with how it looks on my linux machine

I've noticed Windows has this extra 'style' setting in the options that doesn't exist in linux, I've tried all the options in there but also couldn't reproduce it with any of those, so that doesn't seem to effect.
Still, maybe try going through these options and see if it maybe fixes it @FrostedVolcano?
You could also check if Windows Dark/Light mode and these settings are interfering with something? I remember that qbit uses the windows colors/color accents to style the qt elements in the newer releases, maybe something is meddling there with qt?

![qbit_setting](https://github.com/user-attachments/assets/215d46aa-b77c-429e-9f35-d6e0507409ed)


It's a bit odd that the wrong colors in your image are also catppuccin colors, so I don't really know why it would look any different from my setup or has something to do with the window settings, but might be worth a try, otherwise I currently have no other ideas

**Comment by FrostedVolcano** [comment]
Hi @sgoudham and @arminius-smh, 
Thanks for responding and I want to apologize for my late response. I was in fact not using the latest release, I was using the first release. 
What happened was I had downloaded some .qbtheme files from various places a long time ago to try out. I would use one theme for some time and then change to another. Now, I had the mocha.qbtheme from release 1.0.0. After reading your comments I realized that I should be using the latest one, and after doing so now the interface colors are similar to the preview.
Thanks a lot for you help.
